source/stylesheets/site/_banner.scss

```diff
@@ -25,9 +26,13 @@ $_banner-logo-size: $x-large-spacing;
 
 .banner-title {
   font-family: $serif-font-family;
-  font-size: modular-scale(3);
+  font-size: 1.75em;
```

Comment:
Can this be one of the font size variables, e.g. `$large-font-size` or a new one?